src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/TestInfraMultiFileCompilationModuleGroup.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
Can this live in the test project?
I don't see an immediate reason to keep it in the product.